chia/_tests/blockchain/test_blockchain.py

```diff
@@ -1167,14 +1167,14 @@ async def test_bad_pos(self, empty_blockchain: Blockchain, bt: BlockTools) -> No
         await _validate_and_add_block(empty_blockchain, block_bad, expected_error=Err.INVALID_POSPACE)
         block_bad = recursive_replace(
             blocks[-1],
-            "reward_chain_block.proof_of_space.size",
+            "reward_chain_block.proof_of_space.version_and_size",
             32,
         )
         await _validate_and_add_block(empty_blockchain, block_bad, expected_error=Err.INVALID_POSPACE)
         block_bad = recursive_replace(
             blocks[-1],
             "reward_chain_block.proof_of_space.proof",
-            bytes([1] * int(blocks[-1].reward_chain_block.proof_of_space.size * 64 / 8)),
```

Comment:
this test looks confused. the `size` field is the k-value for the plot. It does not indicate how many bytes the proof is.